featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -261,7 +261,7 @@ class Skew(AggregationPrimitive):
     stack_on_self = False
 
     def get_function(self):
-        return 'skew'
+        return pd.Series.skew
```

Comment:
skew doesn't seem to matter as much as count

<img width="1225" alt="Screen Shot 2019-04-17 at 11 36 02 AM" src="https://user-images.githubusercontent.com/371792/56301090-1a91bc00-6105-11e9-896d-6e01d81703a2.png">